test/e2e/kubectl/kubectl.go

```diff
@@ -364,6 +369,86 @@ var _ = SIGDescribe("Kubectl client", func() {
 		})
 	})
 
+	ginkgo.Describe("kubectl get output", func() {
+		ginkgo.It("should contain custom columns for each resource", func() {
+			randString := rand.String(10)
+			ns := "testing-kubectl-get-output-" + randString
+			_, err := c.CoreV1().Namespaces().Create(&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}})
+			framework.ExpectNoError(err)
+			defer deleteNS(c, ns)
```

Comment:
Just use `f.Namespace.Name` which is already creating a random namespace for each test case.

Reply:
Done, thanks!